cfme/utils/appliance/__init__.py

```diff
@@ -23,6 +23,7 @@
 from manageiq_client.api import APIException
 from manageiq_client.api import ManageIQClient as VanillaMiqApi
 from urllib3.exceptions import ConnectionError
+from wait_for import _get_timeout_secs
```

Comment:
This private method is triggering an exception within a test on PRT, why is it being imported?

it looks like its being used to process a `timeout` kwarg, before passing to wait_for as num_sec, can't we just directly pass the timeout and let wait_for process it?

```
>       appliance.wait_for_miq_ready()

cfme/tests/test_appliance.py:443: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
cfme/utils/log.py:177: in newfunc
    return func(*args, **kwargs)
cfme/utils/appliance/__init__.py:1486: in wait_for_miq_ready
    num_secs = _get_timeout_secs(timeout)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

kwargs = 900

    def _get_timeout_secs(kwargs):
>       if "timeout" in kwargs and kwargs["timeout"] is not None:
E       TypeError: argument of type 'int' is not iterable

.cfme_venv/lib64/python3.7/site-packages/wait_for/__init__.py:36: TypeError
```

Reply:
DOH! How could I miss this?

I removed the `_get_timeout_secs` as it is private, but I had to make the `wait_for_miq_ready` accepting `num_sec`, not `timeout`

I did check all usages in pycharm. All uses of `timeout` should be now replaced with `num_sec`